doc/swish/http.tex

```diff
@@ -1134,6 +1137,49 @@ \subsection{WebSocket Protocol}\label{sec:websocket-protocol}
 At compile time, \code{json:write-object} sorts the keys and
 preformats the strings that will separate values.
 
+\defineentry{json:pretty}
+\begin{procedure}
+  \code{(json:pretty \var{x} \opt{\var{op}})}
+\end{procedure}
+\returns{} unspecified
+
+The \code{json:pretty} procedure formats an object \var{x} to the
+optional textual output port \var{op} in a human-readable form. The
+object is indented and keys are sorted using
+\code{natural-string-ci<?}. The default value of \var{op} is
+\code{(current-output-port)}.
+
+Calling \code{(json:pretty x)} is equivalent to calling
+\code{(json:write (current-output-port) x 0)}.
```

Comment:
It's equivalent except that it uses natural-string-ci<? instead of string<?.

Reply:
Good point. Revised to clarify.